features/exporting.feature

```diff
@@ -9,6 +9,8 @@ Feature: Exporting a Journal
         And "tags" in the json output should contain "@idea"
         And "tags" in the json output should contain "@journal"
         And "tags" in the json output should contain "@dan"
+        And entry 1 should have an array called "tags" with 2 elements
```

Comment:
Minor nitpick:

The use of the word "called" in our tests always means that we are invoking something (like the jrnl app itself). Can you take out the word here to avoid confusion?

So, it should just read "entry 1 should have an array "tags" with 2 elements`.